src/bin/pg_autoctl/pgsql.c

```diff
@@ -32,21 +32,36 @@
 #define ERRCODE_OBJECT_IN_USE "55006"
 
 static char * ConnectionTypeToString(ConnectionType connectionType);
+
```

Comment:
Why does this PR changes the white lines here, and in the following lines in this file?